pythonforandroid/recipe.py

```diff
@@ -170,6 +182,11 @@ def versioned_url(self):
             return None
         return self.url.format(version=self.version)
 
+    @property
+    def download_headers(self):
+        key = "DOWNLOAD_HEADERS_" + self.name
+        return environ.get(key, self._download_headers)
```

Comment:
So in that specific case where we have 2 headers, what would the environment variable be like?
`export DOWNLOAD_HEADERS_recipe=?`
Or maybe it's not supported? Could we have the example in the that property docstring?

Reply:
Oh, that's a good point. 

I was trying to keep it generic.  For the environmental variables, the alternative I was thinking about would be to have 

DOWNLOAD_AUTHORIZATION_mypackage = "<github_personal_access_token>"
DOWNLOAD_ACCEPT_mypackage  = "application/vnd.github+json"

**OR**

We can do something like this, which could be parsed:

DOWNLOAD_HEADERS_mypackage = "Authorization=foo,Accept=bar"

What do you think?

Reply:
Another option is to specify the authorization in the following auth token/accept mime type pairs:

DOWNLOAD_AUTHORIZATION_mypackage="token <my_token>,application/vnd.github+json"

Reply:
Ok, I picked a JSON formatted value for the environment variable:

[["header1","foo"],["header2", "bar"]]

https://github.com/kivy/python-for-android/pull/3074/commits/5e14116bf09ca1d48719f7fa7e8c438f1d896ab8